notebooks/working/external_notebook_skeleton.ipynb

```diff
@@ -0,0 +1,219 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Notebook summary\n",
+    "\n",
+    "A paragraph about what we're showing in this notebook, potentially with a table of content that links to each subsection of the notebook. People should be able to read this and understand the flow as well as the main take-away from this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Import packages"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Set work directory and load the dataset. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Description of the dataset -- written or using some descriptive stats"
```

Comment:
I would also suggest that we show a couple of rows of the dataframe. That always helps me understand what the data looks like. Imagine `df.head(5).T`. The transpose is in case the dataframe has many columns (which I believe ours does). Also, can we only show the rows being used in the analysis? I don't think it should matter much, but to be sure, can we not include extraneous rows (e.g. the various household id columns and such)?